language/German/strings.po

```diff
@@ -3098,7 +3125,7 @@ msgstr "Unterstützt: Ja"
 
 #: src/frontend/mame/ui/selmenu.cpp:681
 msgid "System is parent"
-msgstr ""
+msgstr "System ist übergeordnetes System"
```

Comment:
As with the things @darkstar is asking about, is this the terminology used by the German emulation community?  At any rate, it’s inconsistent, as in the filter panel, “Parents” is still translated as “Eltern”.

Reply:
I'll fix it in the remaining locations.